ClientApp/viking-vault/src/components/AccountPage.tsx

```diff
@@ -51,41 +78,101 @@ class AccountPage extends React.Component<any, IAccountState>{
                     if(response.status === 404){
                         this.setState(oldState => {
```

Comment:
You don't need to use the `oldState` kind of calling the `setState` if you don't actually make use of the `oldState` argument. Usually, an unused variable/param is showed as grayed out in Visual Studio Code, this is usually a good sign that you should get rid of that.

In this case just call `setState({ key: value })` like this